modules/rosa-create-objects.adoc

```diff
@@ -861,6 +861,73 @@ Add a machine pool with tags to a cluster.
 $ rosa create machinepool --cluster=mycluster --replicas=2 --instance-type=r5.2xlarge --tags='foo bar,bar baz' --name=mp-1
 ----
 
+[id="rosa-create-network_{context}"]
+== create network
+
+Create a network that creates any necessary AWS resources through AWS CloudFormation templates. This helper command is intended to help create and configure a VPC for use with {hcp-title}. This command also supports zero egress clusters.
+
+[IMPORTANT]
+====
+Running this command creates resources within your AWS account.
+====
+
+[NOTE]
+====
+For custom or advanced configuration, it is highly recommended to use the AWS CLI directly using the `aws cloudformation` command or create a new custom template with the required configurations.
+====
+
+.Syntax
+[source,terminal]
+----
+$ rosa create network [flags]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|<template-name>
```

Comment:
Normally I would have expected this to be marked up with backticks, but, looks like that's not what this CLI reference guide is doing overall in any of the flag/argument/etc tables, so ignoring all of that, but:
```suggestion
|<template_name>
```
Should still use underscores here per user-replaceable variable guidelines, I believe.

Edit: I see the out-of-scope backticks issue already noted via [earlier comment](https://github.com/openshift/openshift-docs/pull/84636#discussion_r1834623832). +1 to that thread / follow-up idea to clean things up.